charts/opentelemetry-operator/values.yaml

```diff
@@ -42,7 +42,7 @@ manager:
     tag: ""
   collectorImage:
     repository: ""
-    tag: 0.113.0
+    tag: 0.114.1
```

Comment:
```suggestion
    tag: 0.114.0
```

Reply:
Version updated in 3c7f10d1e2af474b803714428e141a6cce09e8fb